include/cuco/detail/static_multimap/static_multimap.inl

```diff
@@ -807,7 +807,7 @@ template <uint32_t buffer_size,
           typename OutputIt1,
           typename OutputIt2,
           typename PairEqual>
-__device__ __forceinline__ void
+__device__ inline void
```

Comment:
`__forceinline__` and `inline` are not the same. This change seems out of scope for this PR.

Reply:
Doxygen cannot handle properly `__forceinline__` and it treats it as a return type. So using it together with `void` will make Doxygen complain the doc of return type is missing. I didn't find a way to silence this particular warning. @karthikeyann may have a solution to this.

I noticed before that using `__forceinline__` has little performance impact compared to `inline` thus I was reverting back to `inline` to fix the Doxygen warning.

Reply:
> I noticed before that using __forceinline__ has little performance impact compared to inline thus I was reverting back to inline to fix the Doxygen warning.

I'm surprised that the `__device__` annotation doesn't cause the same problem? 

Can we add an exception to ignore this warning for now? 

That's fine, but I'd like to tackle it as a separate PR where we can study and verify the perf changes in cudf as well.

Reply:
> I'm surprised that the __device__ annotation doesn't cause the same problem?

`__device__` works just fine.

> Can we add an exception to ignore this warning for now?

Now I recall, we can silence this warning by turning off `WARN_NO_PARAMDOC` but I found this is an overkill since it will ignore undocumented parameters or return values which are basically all warnings we want to fix with Doxygen CI check.

Reply:
Oh, I found the solution for this. We should add `__forceinline__` into `PREDEFINED` in the Doxygen config file to specify a custom macro.